### DISCUSSION TITLE
cant get fastf1 to work

### DISCUSSION THREAD

**Discussion by NeizenF** [theOehrly/Fast-F1#282] [General]
i downloaded python 3.9 and the fastf1 files but nothing works. i tried to follow steps but cant manage to. any help pls?

**Comment by theOehrly** [comment]
You don't need to and should not download the files to simply use FastF1. Instead, you should use python's default package installer "pip" to install FastF1. As stated in the readme section, the command for this is `pip install fastf1`. 
It seems like you are new to python, so please try to make yourself familiar with python and its ecosystem by taking a look at some introductory tutorials.
